sdk/identity/azure-identity/out/azure/identity/aio/_credentials/client_credential.pyi

```diff
@@ -0,0 +1,16 @@
+# Stubs for azure.identity.aio._credentials.client_credential (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from ..._base import CertificateCredentialBase, ClientSecretCredentialBase
+from .._authn_client import AsyncAuthnClient
+from azure.core.credentials import AccessToken
+from typing import Any, Mapping
+
+class ClientSecretCredential(ClientSecretCredentialBase):
+    def __init__(self, client_id: str, secret: str, tenant_id: str, **kwargs: Mapping[str, Any]) -> None: ...
+    async def get_token(self, *scopes: str, **kwargs: Any) -> AccessToken: ...
+
+class CertificateCredential(CertificateCredentialBase):
+    def __init__(self, client_id: str, tenant_id: str, certificate_path: str, **kwargs: Mapping[str, Any]) -> None: ...
```

Comment:
Change parameter order - `self, tenant_id, client_id, secret, **kwargs`

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/7951